src/plugins/cloudinary/models/video-source/video-source.const.js

```diff
@@ -1,20 +1,38 @@
 export const DEFAULT_POSTER_PARAMS = { format: 'jpg', resource_type: 'video' };
 
-const DEFAULT_VIDEO_SOURCE_TYPES = ['webm/vp9', 'mp4/h265', 'mp4'];
-
 export const DEFAULT_VIDEO_PARAMS = {
   resource_type: 'video',
   type: 'upload',
   transformation: [],
   sourceTransformation: {},
-  sourceTypes: DEFAULT_VIDEO_SOURCE_TYPES,
+  sourceTypes: ['auto'],
   recommendations: null,
   info: {},
   interactionAreas: {},
   chapters: {}
 };
 
-export const VIDEO_SUFFIX_REMOVAL_PATTERN = RegExp(`\\.(${DEFAULT_VIDEO_SOURCE_TYPES.join('|')})$$`);
+const COMMON_VIDEO_EXTENSIONS = [
+  '3g2',
+  '3gp',
+  'avi',
+  'flv',
+  'm3u8',
+  'ts',
+  'm2ts',
+  'mts',
+  'mov',
+  'mkv',
+  'mp4',
+  'mpeg',
+  'mpd',
+  'mxf',
+  'ogv',
+  'webm',
+  'wmv'
+]; // https://cloudinary.com/documentation/video_manipulation_and_delivery#supported_video_formats
+
+export const VIDEO_SUFFIX_REMOVAL_PATTERN = RegExp(`\\.(${COMMON_VIDEO_EXTENSIONS.join('|')})$$`);
```

Comment:
why do you have to remove the extension? would not the `f_auto:video` take precedence anyway?

Reply:
we always did that, I just updated the list 
https://github.com/cloudinary/cloudinary-video-player/blob/edge/src/plugins/cloudinary/models/video-source/video-source.js#L31

It makes sense, in many cases we need the public ID without the extension, i.e. to get a poster image, what I don't get is why we completely ignore it. I mean, if someone did something like 

```
player.source('elephants.webm');
```

IMO we should treat it as if they did `sourceTypes: ['webm']`

Reply:
ok now i understand that this is just for getting the public id and not for logic related to formats 👍